mne/viz/utils.py

```diff
@@ -1952,7 +1945,7 @@ def _setup_annotation_colors(params):
     raw = params['raw']
     segment_colors = params.get('segment_colors', dict())
     # sort the segments by start time
-    if raw.annotations is not None:
+    if len(raw.annotations) > 0:
```

Comment:
This conditional should no longer be necessary. `numpy.argsort` (and the `ndarray` method wrapper) should function correctly on empty arrays